docs/users.rst

```diff
@@ -170,9 +172,38 @@ be provided, or "pulp_repos" set to true. Otherwise, build will fail.**
 provided, "modules" will be ignored. "pulp_repos" can be used by itself, or with
 either "packages" or "modules".**
 
+.. _container.yaml-autorebuild:
+
+autorebuild
+~~~~~~~~~~~
+
+This section specifies whether and how a build should be rebuilt based on
```

Comment:
I don't think it specifies "whether" -- that comes from a file named ".osbs-repo-config" in the dist-git tree:
```
[autorebuild]
enabled = true|false  # defaults to false
```
ref: https://github.com/projectatomic/osbs-client/blob/0.49/osbs/repo_utils.py#L63